### PR TITLE
Update rnasum input parameters

### PR DESCRIPTION
**Pull request by skanwal**
Updating inputs to bring it at par with the current state of https://github.com/umccr/RNAsum/tree/rnasum-rpackage.

I'll create a new tag that would refer to docker image once we have completed the tests in dev.

### CODE REVIEW

**Review by pdiakumis — CHANGES_REQUESTED** [review]
Following should be removed:

- `clinical_info`
- `clinical_id`
- `hide_code_btn`
- `grch_version`

Following should be added:

- `dragen_mapping_metrics`

Following are in the `arguments` but not the inputs (not sure if adding them in the inputs makes sense but just saying):

- `batch_rm`
- `filter`
- `log`

**Comment by skanwal** [comment]
@pdiakumis - updated in [`e9d5eec` (#480)](https://github.com/umccr/cwl-ica/pull/480/commits/e9d5eec7132984bb5654a47c25de434ed1d0e1d5). Also moved arguments to params and assigned defaults values too.

**Comment by skanwal** [comment]
Tested on ICA with wfr.08d375cb7c24428185142ba8a6b1cb38

File: tools/rnasum/0.5.0/rnasum__0.5.0.cwl
Comment (pdiakumis) [inline tools/rnasum/0.5.0/rnasum__0.5.0.cwl:93]: Thanks for changing this :-)

**Review by pdiakumis — APPROVED** [review]
Thank you, LGTM

**Comment by pdiakumis** [comment]
@skanwal @alexiswl the Actions are erroring out with:

```
Error: couldn't find remote ref update-rnasum-params
```

I take it the PR shouldn't have been merged straight after approval, correct?

**Comment by skanwal** [comment]
Ah yes, that's my bad. We need to wait for GH actions to complete before merging in the PRs in this repo.
@alexiswl - should I reopen the PR?

**Pull request merged**
